standards/rmrk2.0.0/entities/base.md

```diff
@@ -0,0 +1,200 @@
+# Base
+
+A base is a meta-entity, stored on-chain or off chain. It is not an NFT, but can be thought of as
+the interface or abstract class for an NFT's render.
+
+A base is a JSON object with three properties: `id`, `type`, and `parts`. There is an implied
+`owner` field, matching the address that created the base.
+
+Example:
+
+```json=
+{
+    "id": "kanaria_superbird",

```

Comment:
Similar to NFT, should we prepend block number and perhaps something else here to avoid duplicates?

Reply:
Yes, a prefix of block + the word "BASE" somewhere would help us fetch the bases when doing CHANGEISSUER on them too, without needing an extra param in CHANGEISSUER.